app/views/waste_exemptions_engine/shared/_select_address.html.erb

```diff
@@ -1,5 +1,13 @@
-<div class="form-group <%= "form-group-error" if form.errors[:addresses].any? %>">
+<div class="form-group <%= "form-group-error" if form.errors[:addresses].any? || form.errors[:temp_address].any? %>">
   <fieldset id="temp_address">
+    <% if form.errors[:temp_address].any? %>
+      <span class="error-message"><%= form.errors[:temp_address].join(", ") %></span>
```

Comment:
Will there ever be a scenario where there are errors in both `temp_address` and `addresses`? If so, should we be joining them rather than displaying them in separate `span` tags?

Reply:
I actually have no clue about where the `addresses` is coming from 😓so I have left it working for both. Testing it in the normal flow showed that just "temp_address" is a field. Probably some old code was doing that? 🤔 @Cruikshanks can you recall where the `addresses` field is coming from in this context?

Reply:
Because a long, long time ago in a galaxy far, far away the attribute name was `addresses` https://github.com/DEFRA/waste-exemptions-engine/blob/ed44ad18fd23007b02f184a06e8bcd6f26b36903/config/locales/forms/operator_address_forms/en.yml#L23

So feel free to kill it! 😁

Reply:
Alan as confirmed that this is legacy, so removing references to the `addresses` field :)